bmt/README.adoc

```diff
@@ -61,14 +61,20 @@ WFLYJCA0091: -ds.xml file deployments are deprecated. Support may be removed in
 
 HHH000431: Unable to determine H2 database version, certain features may not work
 ----
-
+// Server Distribution Testing
+include::../shared-doc/run-integration-tests-with-server-distribution.adoc[leveloffset=+2]
 // Undeploy the Quickstart
 include::../shared-doc/undeploy-the-quickstart.adoc[leveloffset=+1]
 // Run the Quickstart in Red Hat CodeReady Studio or Eclipse
```

Comment:
this wasn't asked but since changes are needed please remove those 2 includes, such content is outdated

Reply:
sorry but I was referring to removing: 

// Run the Quickstart in Red Hat CodeReady Studio or Eclipse
include::../shared-doc/run-the-quickstart-in-jboss-developer-studio.adoc[leveloffset=+1]
// Debug the Application
include::../shared-doc/debug-the-application.adoc[leveloffset=+1]

Reply:
Oh, sorry. I have just updated it again. Thanks @emmartins